docs/fundamentals/client/third-party-client.md

```diff
@@ -230,6 +230,17 @@ We provide a suite of templates for the Jina Flow, in this [collection](https://
 
 This contribution was made by [Jonathan Rowley](https://jina-ai.slack.com/archives/C0169V26ATY/p1649689443888779?thread_ts=1649428823.420879&cid=C0169V26ATY), in our [community Slack](slack.jina.ai). 
 
+## gRPC
+
+To use the gRPC protocol with another language that python you will need to :
+
+* Download the two proto definition files : `jina.proto` and `docarray.proto` from [github](https://github.com/jina-ai/jina/tree/master/jina/proto) (be sure to use the latest release branch)
+* Compile them with [protoc](https://grpc.io/docs/protoc-installation/) and precise to which programming language you want to compile them.
+* Add the generated files to your project and import them in your code. 
+
+You should finally be able to communicate with your Flow using the gRPC protocol. You can find more information on the gRPC
+`message` and `service` that you can use to communicate in the  [Protobuf documentation](../../proto/docs.md).
```

Comment:
What is this code that they will compile and add to their project? stubs that they can call directly? should we mention how this works or is that outside of our scope here?

Reply:
It is out of the scope I believe. The only way to do it correctly is to 
* have a good proto documentation which explain the stubs that should be use etcc
* have a good how to with another language